src/proto/uopool/uopool.proto

```diff
@@ -107,12 +107,21 @@ message SetReputationResponse {
     SetReputationResult result = 1;
 }
 
+message GetSortedRequest{
+    types.H160 entry_point = 1;
```

Comment:
We could add here how many user operations to return (number/limit).

Reply:
I removed this because current the limit is based on the `max_verification_gas`. The api would return the as much as `max_verification_gas` give. So the limit is not needed.